model/tensorflow/examples/tfdnnc/accuracy.sh

```diff
@@ -4,6 +4,7 @@ dffml accuracy \
   -model-directory tempdir \
   -model-classifications 0 1 2 \
   -model-clstype int \
+  -scorer clfacc \
```

Comment:
- [ ] It might be good to just name this `clf`, since the `acc` is a bit redundant, and isn't present in `mse`